bin/pynani.py

```diff
@@ -11,7 +11,7 @@
 except:
     pass
 
-os.system("cls")
+os.system("cls" if os.name == "nt" else "clear") # supports both Windows and Linux
```

Comment:
I don't recommend clearing the terminal tho.

Reply:
You can remove it.

Reply:
okay